x/vbr/module.go

```diff
@@ -26,8 +26,6 @@ var (
 
 type AppModuleBasic struct{}
 
-var _ module.AppModuleBasic = AppModuleBasic{}
```

Comment:
This line checks that AppModuleBasic implements module.AppModuleBasic at compile-time.

We should keep this for runtime safety sake.

Reply:
At line 24 there is same declaration. I don't think it necessary here.